### DISCUSSION TITLE
Book about unsupervised learning

### DISCUSSION THREAD

**Discussion by Nevermetyou65** [Lightning-AI/dl-fundamentals#84] [General discussions]
Hello Mr. Sebastain,
I am in search of good books about unsupervised learning in DNN. 
Would you be able to recommend any such resources?

**Comment by rasbt** [comment]
Unfortunately I don't have a good recommendation in that area. The more "unsupervised learning" books I read were focused on traditional ML methods and statistics, not DL. I am not sure if there are any good books on that topic out there since it's a rapidly evolving field.
